app/repositories/ActiveSessionRepository.scala

```diff
@@ -33,11 +33,17 @@ import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
 class ActiveSessionRepositoryImpl @Inject()(
-                                             mongo: MongoDriver,
+                                             override val mongo: MongoDriver,
                                              config: Configuration
-                                           )(implicit ec: ExecutionContext) extends ActiveSessionRepository with Logging {
+                                           )(override implicit val ec: ExecutionContext)
+  extends ActiveSessionRepository
+    with IndexManager
+    with Logging {
 
-  private val collectionName: String = "session"
+  override val collectionName: String = "session"
+
+  override val dropIndexes: Boolean =
+    config.get[Boolean]("microservice.services.features.mongo.dropIndexes")
```

Comment:
Have you implemented this in application.conf?

Reply:
it was already there